src/Test/NuGet.Config

```diff
@@ -6,6 +6,7 @@
     <add key="dotnet-core" value="https://www.myget.org/F/dotnet-core/api/v3/index.json" />
     <add key="dotnet-buildtools" value="https://www.myget.org/F/dotnet-buildtools/api/v3/index.json" />    
     <add key="aspnetcidev" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
-    <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />      
+    <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="mstest" value="\\aajohn3\share\packages" />
```

Comment:
Will change this, once the nuget package is released. Will merge only after this is changed.